peachjam/js/components/DocumentContent/pdf-renderer.ts

```diff
@@ -186,44 +187,37 @@ class PdfRenderer {
     pageContainer.dataset.page = String(index + 1);
     pageContainer.classList.add('pdf-content__page');
     pageContainer.style.position = 'relative';
-    pageContainer.appendChild(canvas);
+
     if (this.pdfContentWrapper) {
       this.pdfContentWrapper.appendChild(pageContainer);
     }
 
     // render the page
     await page.render(renderContext).promise;
     // add the text layer
-    pageContainer.appendChild(await this.addTextLayer(page, containerWidth, canvas));
+    pageContainer.append(this.addImageLayer(page, containerWidth, canvas));
```

Comment:
we need to keep `addTextLayer` -- that allows the user to copy and paste text off of the page, and allows us to add text annotations.

So the image should be placed in the div, and then the image layer put on top as-per-usual. So I think you to call both `addTextLayer()` and your new `addImageLayer`.